hack/update-codegen.sh

```diff
@@ -24,9 +24,11 @@ for input in examples/terraform/*/README.md.in; do
   full_target="$dir/$target"
   echo "$full_target"
   cat "$input" > "$dir/$target"
-  $TERRAFORM_DOCS md "$dir" >> "$dir/$target"
+  $TERRAFORM_DOCS --lockfile=false md "$dir" >> "$dir/$target"
```

Comment:
To prevent interference from already locally initialized provider at the developer machine